sdk.d.ts

```diff
@@ -1,247 +1,250 @@
-     */
-    getOrderTransactions(orderID: number): Promise<any>;
-    /**
-     * Get the list of transations for a checkout
-     *
-     * @method
-     * @param {number} checkoutID
-     * @returns {Promise}
-     * @fulfil {object} - The transations list
-     *
-     * @example
-     * const checkoutTransactions = await client.getCheckoutTransactions(123);
-     */
-    getCheckoutTransactions(checkoutID: number): Promise<any>;
-    /**
-     * Verify a webhook alert data using signature and a public key to validate that
-     * it was indeed sent from Paddle.
-     *
-     * For more details: https://paddle.com/docs/reference-verifying-webhooks
-     *
-     * @method
-     * @param  {Object} postData The object with all the parameters sent to the webhook
-     * @return {boolean}
-     *
-     * @example
-     * const client = new PaddleSDK('your-vendor-id', 'your-unique-api-key', 'your-public-key');
-     *
-     * // inside an Express handler which uses express.bodyParser middleware
-     * const isVerified = client.verifyWebhookData(req.body);
-     */
-    verifyWebhookData(postData: any): boolean;
-    /**
-     * Update (upgrade/downgrade) the plan of a subscription
-     *
-     * @method
-     * @param {number} subscriptionID
-     * @param {number} planID
-     * @param {boolean} prorate
-     * @returns {Promise}
-     * @fulfill {object} - The result of the operation
-     *
-     * @example
-     * const result = await client.updateSubscriptionPlan(123);
-     */
-    updateSubscriptionPlan(subscriptionID: number, planID: number, prorate?: boolean): Promise<any>;
-    /**
-     * Update subscription details, quantity, price and or currency
-     *
-     * @method
-     * @param {number} subscriptionID
-     * @param {Object} postData { quantity, price, planID, currency }
-     * @returns {Promise}
-     * @fulfill {object} - The result of the operation
-     *
-     * @example
-     * const result = await client.updateSubscriptionPlan(123, { quantity: 2 });
-     */
-    updateSubscription(subscriptionID: number, postData: any): Promise<any>;
-    /**
-     * Cancels an active subscription
-     *
-     * @method
-     * @param {number} subscriptionID
-     * @returns {Promise}
-     * @fulfil {object} - The result of the operation
-     *
-     * @example
-     * const result = await client.cancelSubscription(123);
-     */
-    cancelSubscription(subscriptionID: number): Promise<any>;
-    /**
-     * Generate a custom pay link
-     *
-     * @method
-     * @param {object} body
-     * @returns {Promise}
-     * @fulfil {object} - The new pay link url
-     *
-     * @example
-     * const custom = await client.generatePayLink({
-     *  "title" : "my custom checkout",
-     *  "custom_message" : "some custom message"
-     * 	"prices": [
-     *		"USD:19.99",
-     *		"EUR:15.99"
-     *	 ]
-     *	});
-     */
-    generatePayLink(body: any): Promise<any>;
-}
+export = PaddleSDK;

```

Comment:
Something is weird with this whole file being rewritten. Try to install the latest dependencies and run `yarn declaration:build`

Reply:
D'oh, I'm on Windows and had CRLF as the line ending, changed it to LF and it's fine now